package.json

```diff
@@ -14,6 +14,7 @@
   },
   "private": true,
   "dependencies": {
+    "@amcharts/amcharts5": "^5.4.4",
```

Comment:
@rrchai The version should be pinned. See https://docs.renovatebot.com/dependency-pinning/ for more info.

Reply:
I will cange to pin the package and run `yarn install` to update lock file.